client/src/components/modal-forms/ManageGraduationForm.js

```diff
@@ -34,18 +34,49 @@ export const ManageGraduationForm = ({ initialValues, onClose, onSubmit }) => {
                     },
                     {
                       value: 'failed_cohort',
-                      label: 'Failed',
+                      label: 'Unsuccessful cohort',
                     },
                   ]}
                 />
                 {
-                  <DependantDateForm
-                    display={values.status === 'post_secondary_education_completed'}
+                  <Field
                     test-id={'editGraduationModalStatus'}
-                    name={'data.graduationDate'}
-                    label={'Graduation Date'}
+                    name={'data.date'}
+                    label={
+                      values.status === 'post_secondary_education_completed'
+                        ? 'Graduation Date'
+                        : 'Unsuccesful Graduation Date'
+                    }
+                    component={RenderDateField}
+                    maxDate={getTodayDate()}
                   />
                 }
+                {values.status === 'failed_cohort' && (
+                  <Field
+                    test-id={'editGraduationModalRehire'}
+                    name='rehire'
+                    component={RenderRadioGroup}
+                    label='I intend on rehiring for this position'
```

Comment:
Struggling to find this question in the designs, what I see is "Will this participant be continuing in the program?" which is very different to a rehire event

What am I missing?

Reply:
@fwpushan here is where the "rehire" language was mistakenly introduced (see my Slack comments)

Reply:
I updated the code and remove all respective `rehire` refs.